doc/neptune-metadata.rst

```diff
@@ -0,0 +1,75 @@
+===========================
+Neptune Metadata Definition
+===========================
+
+This file specifies what fields are in the ``.neptune.yml`` file, what they mean
+and by which tools they are used. The file itself is expected to be in a
+repositories root directory.
+
+.. contents::
+
+library
+-------
+
+Type
+  boolean
+Meaning
+  If set to true, the repository is marked as a library used by other repositories
+Tools
+  - neptune-overview
+  - neptune-autopr
+Example
+  .. code:: yml
+
+    library: true
+
+d2ready
+-------
+
+Type
+  boolean
+Meaning
+  If set to true, the repository is marked as ready for D2 compilation
+Tools:
+  - Makd
+Example
+  .. code:: yml
+
+    d2ready: true
+
+
+support-guarantees
+------------------
+
+Type
+  List
+Meaning
+  Sets the support window and period users can expect for this repository
+Tools:
+  - neptune-overview
+  - neptune-autopr
+Example
+  .. code:: yml
+
+    support-guarantees:
+        - major-months: 6
+        - minor-versions: 2
```

Comment:
Is this really the correct syntax? I thought it was:
```yml
support -guarantees:
  major-months: 6
  minor-versions: 2
```

And I guess the type of this field should be a *mapping*.

http://www.yaml.org/spec/1.2/spec.html#id2759963

Reply:
... I copied that syntax from *your* post :P

Reply:
Is the whitespace in your example just now a mistake? `support -guarantees:`

Reply:
> ... I copied that syntax from your post :P

Haven't you learned anything all these years?!?

> Is the whitespace in your example just now a mistake? support -guarantees:

Yes!

> Another source calls what you have Dictionary

Well, I checked the one and only source, the specs.